ledger/src/blockstore_processor.rs

```diff
@@ -1051,6 +1054,23 @@ fn process_bank_0(
     bank0.freeze();
     blockstore.insert_bank_hash(bank0.slot(), bank0.hash(), false);
     cache_block_meta(bank0, cache_block_meta_sender);
+
+    if let Some(snapshot_config) = snapshot_config {
+        let snapshot_archive_info = solana_runtime::snapshot_utils::bank_to_full_snapshot_archive(
```

Comment:
I wasn't sure if the ordering mattered here, of `blockstore.insert_bank_hash()` and cache_block_meta()` and `bank_to_full_snapshot_archive()`. Is the current spot ok?